src/script/interpreter.cpp

```diff
@@ -982,8 +988,14 @@ bool EvalScript(vector<vector<unsigned char> >& stack, const CScript& script, un
                     }
 
                     // Clean up stack of actual arguments
-                    while (i-- > 1)
+                    while (i-- > 1) {
```

Comment:
formatting { needs to be on next line

Reply:
done...seems that this file never made it into the list of clang formatted files.  I guess you know that, we need a final round of formats for all the subdir's /script /wallet /rpc etc...